packages/animations/example/lib/shared_axis_transition/shared_axis_transition.dart

```diff
@@ -0,0 +1,252 @@
+import 'package:flutter/material.dart';
+import 'package:animations/animations.dart';
+
+/// The demo page for [SharedAxisPageTransitionsBuilder].
+class SharedAxisTransitionDemo extends StatefulWidget {
+  @override
+  _SharedAxisTransitionDemoState createState() {
+    return _SharedAxisTransitionDemoState();
+  }
+}
+
+class _SharedAxisTransitionDemoState extends State<SharedAxisTransitionDemo> {
+  final Key coursePageKey = const Key('Course Page');
```

Comment:
nit: the properties and methods of this class should all start with an underscore.

(I know it doesn't really matter since the entire class is private already, but if for whatever reason this class should ever become public it shouldn't expose these properties)